_clients/javascript.md

```diff
@@ -139,3 +139,21 @@ async function search() {
 
 search().catch(console.log);
 ```
+
+## Circuit Breaker
+
+As part of the client's library of APIs, a `memoryCircuitBreaker` gives you the ability to reject large query responses where the size of the response could crash OpenSearch Dashboards. To set the Circuit Breaker setting, use the [POST _cluster/settings]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cluster-settings/) API operation on your active JS cluster.
+
+`memoryCircuitBreaker` contains two fields:
+
+- `enabled`: A boolean used to turn the circuit breaker on or off. Defaults to `false`.
```

Comment:
In general, capitalize the word Boolean, unless it refers to a specific code element or value that uses the lowercase form. Should "value" follow "Boolean"? Capitalize "Circuit Breaker".